rfc/0023-kubewarden-stack-telemetry.md

```diff
@@ -0,0 +1,338 @@
+|              |                                  |
+| :----------- | :------------------------------- |
+| Feature Name | Kubewarden environment telemetry |
+| Start Date   | 2025-08-28                       |
+| Category     | telemetry                        |
+| RFC PR       | [fill this in after opening PR]  |
+| State        | **TO-BE-DEFINED**                |
+
+# Summary
+
+This RFC describes how the Kubewarden stack will send telemetry data about the
+environment in which it is running. The goal is not to collect information
+about user applications, but rather to learn more about the typical
+environments where Kubewarden is deployed. This data will help guide the
+project's future development.
+
+# Motivation
+
+The Kubewarden team currently has limited visibility into the environments
+where users deploy the project. This information is invaluable for making
+data-driven decisions about the project's future. For this reason, we would
+like to add a feature to the Kubewarden stack that allows sending
+non-sensitive, anonymous environment information to a central service for
+aggregation and analysis
+
+## Examples / User Stories
+
+As the Kubewarden team, I want to know how many PolicyServers and policies are
+typically deployed in an instance.
+
+As the Kubewarden team, I want to know how many policies a PolicyServer usually
+serves.
+
+As the Kubewarden team, I want to know which Kubewarden versions are in use.
+
+As the Kubewarden team, I want to know the most common Kubernetes versions used
+with Kubewarden.
+
+As a Kubewarden user, I want to help improve the project by providing anonymous
+usage data without compromising my privacy or violating data protection
+regulations.
+
+As a Kubewarden user, I want to ensure that sending telemetry data does not
+introduce a significant security vulnerability into my environment.
+
+As a Kubewarden user, I want to ensure that the process of sending telemetry
+data does not consume excessive resources in my environment.
+
```

Comment:
Maybe we can add a user story about a user who doesn't want telemetry data to be sent at all.